course_gather/settings/prod_settings.py

```diff
@@ -18,3 +25,17 @@
         'PORT': int(os.getenv('DB_PORT', '5432'))
     }
 }
+
+# Max-Age cache control
+WHITENOISE_MAX_AGE = 60 * 60
```

Comment:
```suggestion
# Max-Age cache control (in seconds)
WHITENOISE_MAX_AGE = 3600
```

Reply:
Honestly, I'm rather of mixed opinions on this.  `3600` is shorter and cleaner but `60 * 60` (with the inclusion of the comment of course).  Makes this rather easier to read and know instantly how long  it is.

Reply:
^ personally I like the `60 * x` style too. I think as long as it's consistent it's fine.

Reply:
Can you include the (in seconds) comment?